service-swagger-fragment-os/src/main/resources/META-INF/resources/documentation/api/oauth2-redirect.html

```diff
@@ -0,0 +1,74 @@
+<!doctype html>
```

Comment:
Is everything under /src/main/resources/META-INF/resources generated here?  If so, would like to see it not committed.

Reply:
This stuff is not generated and is how they included the OpenAPI UI code in the project. They way they do it here allows REST api projects to replace certain files like the index.html or service-Constants.js which is a function we will take advantage of withe the cohorting engine REST api.